### PR TITLE
Creating True and False question Type

### PR DESCRIPTION
**Pull request by mehtadarshan1**

### CODE REVIEW

File: server/questions.js [hunk 1]
```diff
@@ -130,18 +135,31 @@ var validateQuestionByType = function(question, type){
...
+			result = trueAndFalseValidator(question);
+			break;
+
 		default:
 			break;
 	}
 	return result;
 }
 
 var multipleChoiceValidator = function(question){
```
Comment (esteeroramy) [inline server/questions.js:148]: wasnt this in your previous PR?

**Review by esteeroramy — APPROVED** [review]

**Review by Mohamed-A-Mohamed — APPROVED** [review]
Nice work

File: server/questions.js [hunk 2]
```diff
@@ -130,18 +135,31 @@ var validateQuestionByType = function(question, type){
...
-
-	if (question.choices.length < 2){
-		return 'Need two or more options for Multiple Choice Question';
+	if (question.choices){
```
Comment (ahmedqarmout2) [inline server/questions.js:149]: Combine the 2 ifs into one. If not choices or choices.lengeth more than two. Same for the tf validate

File: server.js [hunk 3]
```diff
@@ -550,8 +558,9 @@ app.get('/question', function(req, res) {
                         return regexForm({studentQuestionForm:true})
                     case common.questionTypes.MULTIPLECHOICE.value:
                         return mcForm({studentQuestionForm:true, question:questionFound})
+                    case common.questionTypes.TRUEFALSE.value:
+                        return mcForm({studentQuestionForm:true, question:questionFound})
                     default:
-                        return res.redirect('/');
```
Comment (ahmedqarmout2) [inline server.js:554]: Should we just a 400 with some error msg.

File: server.js [hunk 4]
```diff
@@ -297,6 +298,11 @@ app.get('/answerForm', function(req, res){
                 common.questionTypes.MULTIPLECHOICE.template,{
                 answerForm:true});
             break;
+        case common.questionTypes.TRUEFALSE.value:
+            res.status(200).render(
+                common.questionTypes.TRUEFALSE.template,{
+                answerForm:true});
+            break;
         default:
             return res.redirect('/');
```
Comment (ahmedqarmout2) [inline server.js:307]: Same as below. Send 400 with some error statement

File: server.js [hunk 5]
```diff
@@ -550,8 +558,9 @@ app.get('/question', function(req, res) {
                         return regexForm({studentQuestionForm:true})
                     case common.questionTypes.MULTIPLECHOICE.value:
                         return mcForm({studentQuestionForm:true, question:questionFound})
+                    case common.questionTypes.TRUEFALSE.value:
+                        return mcForm({studentQuestionForm:true, question:questionFound})
```
Comment (ahmedqarmout2) [inline server.js:562]: tf form not Mc form

File: server.js (hunk 4 above)
Comment (mehtadarshan1) [reply server.js:307]: how about 404?

File: server.js (hunk 5 above)
Comment (mehtadarshan1) [reply server.js:562]: good catch

**Pull request merged**